### ISSUE TITLE
Validation error if amp-list references an template

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amphtml#2517]
The following sample results in a validation errors (but works):

```
<amp-list width=300
      height=100
      layout=responsive
      src="https://ampbyexample.com/json/examples.json"
      template="amp-template-id">
  </amp-list>
```

Validation error:
```
validator.js:330 http://localhost:8000/components/amp-list/:1682:12 The attribute 
'template' may not appear in tag 'amp-list'. (see https://www.ampproject.org/docs/reference/extended/amp-list.html)
```

According to the docs the template attribute is supported. 

Example page: https://ampbyexample.com/components/amp-list/?development=1

//cc @gregable @dvoytenko

**Comment by dvoytenko** [comment]
@Gregable PTAL. `template` attribute should be allowed as per https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/amp-list.md#usage

**Comment by rudygalfi** [comment]
@Gregable Have you had a chance to look at this?

**Comment by Gregable** [comment]
Will have this in the next release.

**Comment by rudygalfi** [comment]
Closed via #2711.

**Comment by Gregable** [comment]
FYI, this will be visible on the cdn.ampproject.org validator in about an hour.
